src/azure-cli/azure/cli/command_modules/aro/commands.py

```diff
@@ -0,0 +1,25 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands import CliCommandType
+from azure.cli.command_modules.aro._client_factory import cf_aro
+from azure.cli.command_modules.aro._format import aro_show_table_format
+from azure.cli.command_modules.aro._format import aro_list_table_format
+from azure.cli.command_modules.aro._help import helps  # pylint: disable=unused-import
+
+
+def load_command_table(self, _):
+    aro_sdk = CliCommandType(
+        operations_tmpl='azure.cli.command_modules.aro.vendored_sdks.azure.mgmt.redhatopenshift.v2020_04_30.operations#OpenShiftClustersOperations.{}',  # pylint: disable=line-too-long
+        client_factory=cf_aro)
+
+    with self.command_group('aro', aro_sdk, client_factory=cf_aro, is_preview=True) as g:
+        g.custom_command('create', 'aro_create', supports_no_wait=True)
+        g.custom_command('delete', 'aro_delete', supports_no_wait=True, confirmation=True)
+        g.custom_command('list', 'aro_list', table_transformer=aro_list_table_format)
```

Comment:
If ARO supports wait option, we highly recommend you provide command `wait` to give user the ability to wait until a desired operation is fine.
Please refer to https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md

Reply:
Added